packages/css/src/components/header/header.scss

```diff
@@ -13,29 +13,40 @@
   * Display grid is used to let both branding sections overlap.
   */
   display: grid;
+  font-family: var(--ams-header-font-family);
   padding-block: var(--ams-header-padding-block);
   padding-inline: var(--ams-header-padding-inline);
 }
 
-.ams-header__branding {
+.ams-header__logo-link {
   align-items: center;
   align-self: start; // To align the branding section to the top of the header when it wraps
-  column-gap: var(--ams-header-branding-column-gap);
-  display: flex;
+  column-gap: var(--ams-header-logo-link-column-gap);
+  display: inline-flex;
   grid-area: 1 / 1; // To allow this section to overlap with the second branding section
+  outline-offset: var(--ams-header-logo-link-outline-offset);
+  text-decoration: none;
 }
 
-.ams-header__branding--hidden {
+.ams-header__logo-link--hidden {
   opacity: 0%;
   user-select: none; // The hidden branding section should not be selectable
 }
 
-.ams-header__logo-link {
-  outline-offset: var(--ams-header-logo-link-outline-offset);
+.ams-header__logo-container {
```

Comment:
Don’t these styles work directly on the svg?

Reply:
No, the SVG has a `viewBox`, which makes it responsive. Setting the `inline-size` shrinks the logo, it doesn't cut it off. And we need that responsive behaviour because the logo grows in size in wider screens.